src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/RazorFileChangeDetectorTest.cs

```diff
@@ -102,7 +102,7 @@ public void FileSystemWatcher_RazorFileEvent_Background_AddRemoveDoesNotNotify()
             listener.Setup(l => l.RazorFileChanged(filePath, It.IsAny<RazorFileChangeKind>())).Callback(() => listenerCalled = true);
             var fileChangeDetector = new RazorFileChangeDetector(Dispatcher, FilePathNormalizer, new[] { listener.Object })
             {
-                EnqueueDelay = 50,
+                EnqueueDelay = 10,
```

Comment:
Was this not working properly before, or is switching to `10` just a test optimization?

Reply:
Ah woops, I should have removed this change (it's unrelated). Doesn't have any functional impact in the test. Actually makes them run a bit faster 😆

Reply:
I guess given it's just a speed improvement / doesn't have negative drawbacks I'll leave it in for now. SOrry for the unrelated change though!